assets/styles/delivery/layout/delivery.scss

```diff
@@ -14,13 +14,18 @@ body {
 
   .card-title {
     text-overflow: ellipsis;
-    overflow: hidden;
+    overflow: auto;
     white-space: nowrap;
   }
 
   .fade-text {
-    height: 4em;
-    overflow: scroll;
+    overflow: auto;
+    max-height: 10vh;
```

Comment:
is em or vh used more? think we should stick the most commonly used one - either way looks great :)

Reply:
Honestly just thought about setting a max-height instead of a fixed one, and carried over the vh usage from another authoring style. 

I did use `vh` here though, because I also set the image in `vh`. 
Good thing with `vh/wh` in this cases is that it does take in account the side/scroll bars into the viewport total, unlike just setting a percentage. And we have many places (like this and the sections index) that are using scrolling a lot.(which I don't love lol). So I think in this case I would rather go for % instead if not vh.

Ultimately, for those places I believe we need to refactor all the elements into using more relative (at least %) sizing rather than fixed heights/widths and relying in scrolling if content doesn't fit.

Not the most styling-savvy so I can change these into em if it makes more sense :)